docs/advanced/Channels.md

```diff
@@ -293,3 +293,90 @@ In the above example, we create a channel using the `channel` factory. We get ba
 The `watchRequests` saga then forks three worker sagas. Note the created channel is supplied to all forked sagas. `watchRequests` will use this channel to *dispatch* work to the three worker sagas. On each `REQUEST` action the Saga will put the payload on the channel. The payload will then be taken by any *free* worker. Otherwise it will be queued by the channel until a worker Saga is ready to take it.
 
 All the three workers run a typical while loop. On each iteration, a worker will take the next request, or will block until a message is available. Note that this mechanism provides an automatic load-balancing between the 3 workers. Rapid workers are not slowed down by slow workers.
+
+## Using the `multicastChannel` to communicate with *different* workers
+
+In the section above we saw how `channels` can be used to load-balance requests between the *same* worker *forked* multiple times. What if we needed to `put` an action to a channel and have *several different* workers consuming it?
+
+We may need to pass the incoming request to different workers to perform different side effects.
+
+Here an example using `channels` where we can see the problem: when we `put` in the `channel` with `yield put(chan, payload)` we will always run only *one* worker (`logWorker` or `mainWorker`), not *both*.
+
+```javascript
+import { channel } from 'redux-saga'
+import { take, fork, call, put } from 'redux-saga/effects'
+
+function* watchRequests() {
+  // create a channel to queue incoming requests
+  const chan = yield call(channel)
+
+  // fork both workers
+  yield fork(logWorker, chan)
+  yield fork(mainWorker, chan)
+
+  while (true) {
+    const { payload } = yield take('REQUEST')
+
+    // put here will reach only one worker, not both!
+    yield put(chan, payload)
+  }
+}
+
+function* logWorker(channel) {
+  while (true) {
+    const payload = yield take(channel)
+    // Log the request somewhere..
+    console.log('logWorker:', payload)
+  }
+}
+
+function* mainWorker(channel) {
+  while (true) {
+    const payload = yield take(channel)
+    // Process the request
+    console.log('mainWorker', payload)
+  }
+}
+```
+
+To solve the problem we need to use the `multicastChannel` that will *broadcast* the action to all the workers simultaneously.
+> Note that using `take` with the `multicastChannel` requires us to pass the additional argument *pattern* - that you can use to filter the actions to `take`.
```

Comment:
It is indeed true:
https://github.com/redux-saga/redux-saga/blob/24e9a68d621fd2a57a29b3b80e4b54ecb22fa593/packages/core/src/internal/io.js#L38-L40

Although, in retrospect, it seems like a weird limitation. This should only be a requirement for `stdChannel`.